static/js/Misc.jsx

```diff
@@ -3300,6 +3300,17 @@ const Autocompleter = ({getSuggestions, showSuggestionsOnSelect, inputPlaceholde
     )
 }
 
+const ImageWithCaption = ({photoLink, enCaption, heCaption }) => {
```

Comment:
I would change the caption props to be `caption` that is an object with keys `en` and `he` to match the way we generally pass bilingual data.

Reply:
I totally agree, that makes a lot more sense. Thanks Noah!